packages/reactotron-core-client/src/reactotron-core-client.ts

```diff
@@ -161,6 +177,12 @@ export class Client {
 
       // trigger our plugins onCommand
       forEach(plugin => plugin.onCommand && plugin.onCommand(command), this.plugins)
+
+      // trigger out register custom commands
+      if (command.type === "custom") {
+        // TODO: Ramda-tize
```

Comment:
I prefer this way.  I've fallen of the `ramda` train for straight-forward statements like this.  💯

Reply:
:+1: